e2e/testcases/stress_test.go

```diff
@@ -261,6 +265,91 @@ func TestStressLargeRequest(t *testing.T) {
 	}
 }
 
+func TestStress100CRDs(t *testing.T) {
```

Comment:
The indentation needs to be fixed.

Reply:
what's wrong with the indentation? Running `make fmt-go` doesn't yield any issue

Reply:
![image](https://github.com/GoogleContainerTools/kpt-config-sync/assets/57464258/86c9c2e1-f712-4c24-9a37-543a7a9bd304)

Reply:
I'm not seeing the same indentation in my side screen/8U8jBpSxF4wvYFy, could this be due different github settings https://stackoverflow.com/questions/70774875/problems-with-indentations-on-github

Reply:
Restart the browser, and it looks good now.